packages/core/src/render3/instructions.ts

```diff
@@ -953,14 +956,13 @@ export function viewCreate(viewBlockId: number): boolean {
  *
  * @param viewIndex The index of the view's static data in containerStatic
  */
-function setNgStaticDataForView(viewIndex: number): void {
-  ngDevMode && assertNodeType(previousOrParentNode.parent !, LNodeFlags.Container);
-  const containerStatic =
-      (previousOrParentNode.parent !.staticData as LContainerStatic).containerStatic;
+function initViewStaticData(viewIndex: number, parent: LContainer): NgStaticData {
```

Comment:
Start the function description with something like 
"Initialize the static data for the active view."?

Add the missing `@param` and `@returns` clause to the JsDoc